spec/models/recording_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Recording, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Move assertions from `spec/models/recording.rb` to here and delete that file (it won't run because it doesn't end in _spec)!